# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux24-psionw/gcc-3.4-upstream.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- linux-2.4.27/drivers/char/keyboard.c.vanilla	2005-02-01 23:05:43.391427456 +0100
+++ linux-2.4.27/drivers/char/keyboard.c	2005-02-01 23:06:31.297144680 +0100
@@ -419,6 +419,7 @@
 	}
 	rep = 0;
 out:
+	;
 }
 
 /*
--- linux-2.4.27/include/asm-arm/hardware/psionw.h.vanilla	2005-02-01 23:04:22.173774416 +0100
+++ linux-2.4.27/include/asm-arm/hardware/psionw.h	2005-02-01 23:04:28.740776080 +0100
@@ -399,4 +399,4 @@
 #define COEOI		(0x0a0c)	/* Codec end of interrupt, write only */
 #define COTEST		(0x0a10)	/* Codec test register */
 
-#endif /* __ASM_HARDWARE_PSIONW_H */					\
+#endif /* __ASM_HARDWARE_PSIONW_H */
--- linux-2.4.27/drivers/char/keyboard_psion.c.vanilla	2005-02-01 23:14:21.000000000 +0100
+++ linux-2.4.27/drivers/char/keyboard_psion.c	2005-02-01 23:14:43.000000000 +0100
@@ -54,6 +54,8 @@
     "\000" "\000" "\000" ","	"\000" " "    "\000";
 #endif
 
+void debug_gpio(void);
+
 void psion_cleartable(void) 
 {
 	int key;
